metals/src/main/scala/scala/meta/internal/troubleshoot/ProblemResolver.scala

```diff
@@ -19,7 +19,8 @@ class ProblemResolver(
     workspace: AbsolutePath,
     mtagsResolver: MtagsResolver,
     currentBuildServer: () => Option[BspSession],
-    javaHome: () => Option[String]
+    javaHome: () => Option[String],
+    isTestExplorerProvider: () => Boolean
```

Comment:
I'm not a fan of passing boolean flags, but it was easier than passing whole `clientConfig`